app/models/member.rb

```diff
@@ -0,0 +1,4 @@
+class Member < ApplicationRecord
+  has_one_attached :image
+  acts_as_paranoid
```

Comment:
Remember to implement [Discard gem](https://github.com/jhawthorn/discard) for soft delete.

Reply:
Don't forget to add any validation to your model. Any constraint you have in the database table needs the same into the model.

Reply:
updated